fishtest/fishtest/rundb.py

```diff
@@ -109,16 +103,19 @@ def new_run(self, base_tag, new_tag, num_games, tc, book, book_depth,
     if spsa is not None:
       run_args['spsa'] = spsa
 
+    [tc_base, tc_inc] = tc.split('+')
```

Comment:
unfortunately there are tc written as "20", "20/5" and some typo/garbage as "20+0.0.5", "20+20+0.5", "inf"

Reply:
thanks, i've updated this logic to deal with the garbage values. we should add validations on the runs documents later to prevent garbage from getting in there